aocd_example_parser/extraction.py

```diff
@@ -6,6 +6,46 @@
 from aocd.examples import Page
 
 
+from aocd.examples import Example
+from aocd.examples import Page
+
+from .extraction import extract_examples
```

Comment:
you don't need to import it when it's in the same module

Reply:
fixed.